ouroboros-consensus-cardano/tools/db-analyser/Main.hs

```diff
@@ -254,29 +250,67 @@ processAll immDB rr callback = do
 -------------------------------------------------------------------------------}
 
 data CmdLine = CmdLine {
-      clConfig    :: FilePath
+      clConfig    :: [FilePath]
     , clIsMainNet :: Bool
     , clImmDB     :: FilePath
+    , clBlockType :: BlockType
     , clAnalysis  :: AnalysisName
-    }
+    } deriving Show
+
+data BlockType = Byron | Shelley | Cardano | CheckDBMarker
```

Comment:
I was a bit surprised when I saw `CheckDBMarker`, but after looking around, I realised what it meant: document its meaning. Alternatively, omit it, as it isn't really a `BlockType` and use `Nothing` for this case.